app/usecases/email.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+import smtplib
+import ssl
+from email.mime.application import MIMEApplication
+from email.mime.message import MIMEMessage
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.utils import COMMASPACE
+from email.utils import formatdate
+
+import app.settings
+
+
+def send_email(
+    receivers: list[str],
+    subject: str,
+    content,
+    attachments: list[MIMEMessage],
+) -> None:
+    """Sends an email using the specified SMTP settings in the config with the specified subject, content and attachments to the specified receivers"""
+
+    with smtplib.SMTP_SSL(
```

Comment:
this lib is not async so will block the entire server whenever gdpr is requested, can we change library here?

Reply:
Haven't thought about that, there seems to be aiosmtplib

Reply:
pls move to that

Reply:
Isn't it already async since I do `asyncio.create_task(app.usecases.gdpr.send_gdpr_email(ctx.player))`?

Reply:
that runs that function in async but the actual smtp lib is not async, so it is blocking

Reply:
I see